Dataset contrail detection

leothomas opened the pull request:
### What I did: 
Added contrail detection dataset to the COVID API backend

### How I did: 
Added a metadata file in `/dataset/static/` with contrail info. The background tile (`MODIS_Terra_CorrectedReflectance_TrueColor`) gets loaded from the GIBS API. 

### How you can test it: 
Pull this PR, run `docker-compose up --build` and direct your browser to [localhost:8000/v1/datasets](http://localhost:8000/v1/datasets) and look for the `detections-contrail` dataset. 

You can also run the front end (pointing the API to your localhost). 

# Note: 
This PR is in draft mode as there are two things to address: 
- [ ] Need to add the "info" section of the dataset metadata (+ any styling items) @olafveerman 
- [ ] ~Need to figure out an issue with the front end which fails to render the detections geojson due to the following error:~
```
Error: Unknown Geometry Type
    at Object.coordEach (index.js:131)
    at bbox (bbox:20)
    at types.js:217
``` 
@danielfdsilva 

### Screen shot: 
This is what the MODIS tile loaded from the GIBS API looks like
![image](https://user-images.githubusercontent.com/11858457/101839859-194ea780-3b11-11eb-8d9e-0c35aefae6ec.png)

olafveerman commented:
@leothomas Pushed some minor changes in the latest commit.

Contrail detections weren't loaded at all in the frontend, causing it to throw `Error: Unknown Geometry Type` when trying to zoom to the (non-existent) bbox.

leothomas commented:
Great catch @olafveerman. It also turned out I had some problems with the projection of the background dataset which I've fixed: 
![image](https://user-images.githubusercontent.com/11858457/102831483-ca223580-43a0-11eb-8317-bf4200a8a574.png)


I'm waiting on Iksha to validate the detections. There's one issue left to address: I was originally using the geographic 
projection (`epsg4326`) of the `MODIS_Terra_CorrectedReflectance_TrueColor`, which is why the detections were not being displayed over the correct background tiles. I switched to using the web-mercator projection (`epsg3857: https://gibs.earthdata.nasa.gov/wmts/epsg3857/best/MODIS_Terra_CorrectedReflectance_TrueColor/default/{date}/GoogleMapsCompatible_Level9/{z}/{y}/{x}.jpeg`), however this for some reason, the `epsg3857` version doesn't accept dates in the `yyyy_MM_dd` format and instead requires the date to be `yyyy-MM-dd`. 

Eg: 
[4326 tile with `yyyy_MM_dd`](https://gibs.earthdata.nasa.gov/wmts/epsg4326/best/MODIS_Terra_CorrectedReflectance_TrueColor/default/2020_03_05/250m/5/10/10.jpg) --> works ✅ 

[3857 tile with `yyyy_MM_dd`](https://gibs.earthdata.nasa.gov/wmts/epsg3857/best/MODIS_Terra_CorrectedReflectance_TrueColor/default/2020_03_05/GoogleMapsCompatible_Level9/5/10/10.jpg) --> fails ❌ 

[3857 tile with `yyyy-MM-dd`](https://gibs.earthdata.nasa.gov/wmts/epsg3857/best/MODIS_Terra_CorrectedReflectance_TrueColor/default/2020-03-05/GoogleMapsCompatible_Level9/5/10/10.jpg) --> works ✅ 

Since the backend url is pre-populated and sent to the front end to fill in `{date}` and `{z},{x},{y}` values there needs to be some handling on the front end to format the date with hyphens instead of underscores in the case of detections contrails

leothomas commented:
@reviewers: 
I added `RUN sed -i 's/DEFAULT@SECLEVEL=2/DEFAULT@SECLEVEL=1/' /etc/ssl/openssl.cnf` to the `ecs/Dockerfile` (see [this stackoverflow comment](https://stackoverflow.com/questions/61631955/python-requests-ssl-error-during-requests#comment112879465_61643770). I'm not 100% of potential "side effects" this might have from a security standpoint. I would appreciate a cautious 👀  at that! Thanks!
